src/main/java/org/jenkinsci/plugins/github/admin/GitHubDuplicateEventsMonitor.java

```diff
@@ -0,0 +1,75 @@
+package org.jenkinsci.plugins.github.admin;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import hudson.Extension;
+import hudson.model.AdministrativeMonitor;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.github.Messages;
+import org.jenkinsci.plugins.github.webhook.subscriber.DuplicateEventsSubscriber;
+import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.stapler.WebMethod;
+import org.kohsuke.stapler.json.JsonHttpResponse;
+import org.kohsuke.stapler.verb.GET;
+import net.sf.json.JSONObject;
+
+@SuppressWarnings("unused")
+@Extension
+public class GitHubDuplicateEventsMonitor extends AdministrativeMonitor {
+
+    @VisibleForTesting
+    static final String LAST_DUPLICATE_CLICK_HERE_ANCHOR_ID = GitHubDuplicateEventsMonitor.class.getName()
+                                                              + "#last-duplicate";
```

Comment:
Is a string containing `#` even a legitimate anchor name? Seems unusual at best.
```suggestion
    static final String LAST_DUPLICATE_CLICK_HERE_ANCHOR_ID = GitHubDuplicateEventsMonitor.class.getName()
                                                              + ".last-duplicate";
```
?

Reply:
makes sense, using `#` was incorrect that it may clash with CSS as well. (apparently it works as also currently working in the test)